substreams-helper/src/price.rs

```diff
@@ -0,0 +1,52 @@
+use std::ops::Div;
+use std::str::FromStr;
+
+use bigdecimal::BigDecimal;
+use serde::Deserialize;
+use substreams::Hex;
+
+use crate::{math, rpc};
+
+pub enum Network {
```

Comment:
Should probably put shared enums in a separate module

Reply:
By a shared module do you mean something like types.rs? Not doing that because it is not shared yet, will move to a shared module when it has to

Reply:
well this is an enum that we know for sure will be shared